btrdb/stream.py

```diff
@@ -1107,13 +1107,19 @@ def clone(self):
                 setattr(clone, attr, deepcopy(val))
         return clone
 
-    def windows(self, width, depth):
+    def windows(self, *args, **kwargs):
```

Comment:
I think the original version (using specific args, with expressive variable names) was clearer to understand and use, and simplified the code since it didn't need the if/else statements below. Is there a way we can implement the functionality you want without using *args & *kwargs?